src/common/options.cc

```diff
@@ -4622,7 +4622,7 @@ std::vector<Option> get_global_options() {
     .set_description(""),
 
     Option("filestore_merge_threshold", Option::TYPE_INT, Option::LEVEL_DEV)
-    .set_default(10)
+    .set_default(-10)
```

Comment:
should update doc/rados/configuration/filestore-config-ref.rst too - unfortunately this isn't auto-generated from options.cc